internal/oc/metrics.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package oc supports OpenCensus tracing and metrics for the Go Cloud Development Kit.
+package oc
+
+import (
+	"fmt"
+
+	"go.opencensus.io/plugin/ocgrpc"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// LatencyMeasure returns the measure for method call latency used
+// by Go CDK APIs.
+func LatencyMeasure(pkg string) *stats.Float64Measure {
+	return stats.Float64(
+		pkg+"/roundtrip_latency",
```

Comment:
Same as other comment, not sure `roundtrip` is right here.

Reply:
Done.